ext/zend_test/test_arginfo.h

```diff
@@ -1,6 +1,9 @@
 /* This is a generated file, edit the .stub.php file instead.
  * Stub hash: dd3f852ea9f8e3a356ed226380edf5cc336f8a4e */
 
+ZEND_STATIC_ASSERT(PHP_VERSION_ID >= 80000, "test_arginfo.h only supports PHP version ID 80000 or newer, "
```

Comment:
Are you fine with added such static asserts for arginfo files when the stub file has a version constraint?

Reply:
I don't have any strong opinions about this

Reply:
OK, then I merge it, and in case anyone complains, I get rid of it or change it.